x-pack/plugins/alerting/server/alerts_client/lib/build_updated_recovered_alert.test.ts

```diff
@@ -161,7 +162,6 @@ describe('buildUpdatedRecoveredAlert', () => {
         timestamp: '2023-03-29T12:27:28.159Z',
       })
     ).toEqual({
-      ...alertRule,
```

Comment:
`alertRule` is an object with "flattened" properties: https://github.com/elastic/kibana/blob/043425421393e26779de81a1b1f1b188fb40f9a6/x-pack/plugins/alerting/server/alerts_client/lib/test_fixtures.ts#L60-L72

But is being replaced with non-flattened properties, below.

I'm thinking this is ok because we always end up flattening, in the end?

Reply:
Yea, essentially!